pytorch_lightning/accelerators/dp_accelerator.py

```diff
@@ -172,3 +174,10 @@ def reinit_scheduler_properties(self, optimizers: list, schedulers: list):
                 scheduler.__class__.__mro__[idx].__init__(scheduler, optimizer)
                 if state is not None:
                     scheduler.load_state_dict(state)
+
+    @property
+    def reference_model(self) -> LightningModule:
```

Comment:
this is confusing as here it is the property and above is method

Reply:
this is a mistake, let me fix